src/tests/number.test.ts

```diff
@@ -38,4 +38,16 @@ describe('number module', () => {
       assert.strictEqual(result, 20)
     })
   })
+
+  describe('round function', () => {
+    test('handles undefined', () => {
+      const result = _.round(12)
+      assert.strictEqual(result, 12)
+    })
+
+    test('rounds 20.456  with precision 2', () => { 
+      const result = _.round(20.456, 2);
+      assert.strictEqual(result, 20.46);
+    })
```

Comment:
I'd love to see a few more test cases here that cover common rounding at different precisions. Just a few
- `0.005` at precision `1`?
- `19.9999` at precision `4`?
- `2.5` at precision `9`

It might seem a bit dull but when it comes to rounding it could impact finance/cost/price so better very very safe than sorry.